src/zkbob/ZkBobPool.sol

```diff
@@ -42,6 +44,11 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Zk
     address public immutable token;
     IZkBobDirectDepositQueue public immutable direct_deposit_queue;
 
+    uint256[3] private __deprecatedGap;
+    IEnergyRedeemer public redeemer;
+    IZkBobAccounting public accounting;
+    uint96 public pool_index;
```

Comment:
Should it be mentioned in the migration procedures notes that `pool_index` must be initiated to proper value?

Reply:
Thanks, added an init logic for pool_index, missed it on the first iteration.